commons/zenoh-shm/src/api/client_storage/mod.rs

```diff
@@ -31,17 +31,16 @@ use crate::{
     reader::{ClientStorage, GlobalDataSegmentID},
 };
 
-lazy_static! {
-    /// A global lazily-initialized SHM client storage.
-    /// When initialized, contains default client set,
-    /// see ShmClientStorage::with_default_client_set
-    #[zenoh_macros::unstable_doc]
-    pub static ref GLOBAL_CLIENT_STORAGE: Arc<ShmClientStorage> = Arc::new(
-        ShmClientStorage::builder()
-            .with_default_client_set()
-            .build()
-    );
-}
+#[dynamic(lazy, drop)]
+/// A global lazily-initialized SHM client storage.
+/// When initialized, contains default client set,
+/// see ShmClientStorage::with_default_client_set
```

Comment:
Nitpick here. If you want to reference the actual function/type in the doc you can use markdown-like syntax `[`with_default_client_set`](ShmClientStorage::with_default_client_set)`

Reply:
fixed